packages/router/src/adapters/subgraph/graphqlsdk.ts

```diff
@@ -889,41 +882,56 @@ export const GetAssetBalanceDocument = gql`
+export function getSdk(client: FallbackSubgraph, withWrapper: SdkFunctionWrapper = defaultWrapper) {
   return {
     GetSenderTransactions(variables: GetSenderTransactionsQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetSenderTransactionsQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetSenderTransactionsQuery>(GetSenderTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetSenderTransactions');
+      return withWrapper((wrappedRequestHeaders) => client.request<GetSenderTransactionsQuery>(GetSenderTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), "GetSenderTransactions");
     },
     GetReceiverTransactions(variables: GetReceiverTransactionsQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetReceiverTransactionsQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetReceiverTransactionsQuery>(GetReceiverTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetReceiverTransactions');
+      return withWrapper((wrappedRequestHeaders) => client.request<GetReceiverTransactionsQuery>(GetReceiverTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), "GetReceiverTransactions");
     },
     GetTransaction(variables: GetTransactionQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetTransactionQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionQuery>(GetTransactionDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetTransaction');
+      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionQuery>(GetTransactionDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), "GetTransaction");
     },
     GetTransactions(variables?: GetTransactionsQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetTransactionsQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionsQuery>(GetTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetTransactions');
+      return withWrapper((wrappedRequestHeaders) => client.request<GetTransactionsQuery>(GetTransactionsDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), "GetTransactions");
     },
     GetAssetBalance(variables: GetAssetBalanceQueryVariables, requestHeaders?: Dom.RequestInit["headers"]): Promise<GetAssetBalanceQuery> {
-      return withWrapper((wrappedRequestHeaders) => client.request<GetAssetBalanceQuery>(GetAssetBalanceDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), 'GetAssetBalance');
+      return withWrapper((wrappedRequestHeaders) => client.request<GetAssetBalanceQuery>(GetAssetBalanceDocument, variables, {...requestHeaders, ...wrappedRequestHeaders}), "GetAssetBalance");
```

Comment:
MARK: Real change in this file is here. All of the above is just Prettier eslint formatting